<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>583</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QWidget" name="ClockPanel" native="true">
        <property name="minimumSize">
         <size>
          <width>340</width>
          <height>325</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="clockLayout"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="clockFeedPanel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Подведение часов</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeEdit">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="displayFormat">
            <string>HH:mm:ss</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pb_setFeed">
             <property name="text">
              <string>Задать</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pb_closeWindow">
             <property name="text">
              <string>Выход</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBar">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="actionColor_options"/>
    <addaction name="actionClock_feed"/>
    <addaction name="actionTable"/>
   </widget>
   <addaction name="menuBar"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionColor_options"/>
   <addaction name="actionClock_feed"/>
   <addaction name="actionTable"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>173</width>
     <height>140</height>
    </size>
   </property>
   <property name="windowIcon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/Img/calendar.png</normaloff>:/new/Img/calendar.png</iconset>
   </property>
   <property name="windowTitle">
    <string>Таблица подводов</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>155</width>
       <height>336</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <column/>
    </widget>
    <widget class="QPushButton" name="pb_clear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>155</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionColor_options">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/Img/colors.png</normaloff>:/new/Img/colors.png</iconset>
   </property>
   <property name="text">
    <string>Настройка цвета</string>
   </property>
  </action>
  <action name="actionClock_feed">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/Img/auto-repair.png</normaloff>:/new/Img/auto-repair.png</iconset>
   </property>
   <property name="text">
    <string>Подвод часов</string>
   </property>
  </action>
  <action name="actionTable">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/Img/calendar.png</normaloff>:/new/Img/calendar.png</iconset>
   </property>
   <property name="text">
    <string>Таблица подводов</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
